src/main/resources/META-INF/plugin.xml

```diff
@@ -10,6 +10,17 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>0.7.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/214">As a user I want to be able to open a terminal in a pod</a></li>
```

Comment:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/214">Open Terminal Shell for running containers</a></li>
```

Reply:
@mohitsuman this is not explanatory text. These are the changelog items. There's nothing to correct here.

Reply:
There's nothing to change here. These are the changelog items which are copy/pasted from the github issues.

Reply:
Yeah but this is what gets displayed on the landing page. So it need not be the GitHub issue same wordings. We need to add wording which explains what functionality it supports now, given it already has an issue link for folks to read more.

Reply:
ideally these would be generated automatically, so I'd be cautious with changes. If changes, then those should be done to the github items like we do ex. in tekton where it's required to prefix an issue with "fix:", "feat:", etc. (see here: https://github.com/redhat-developer/intellij-tekton/commits/main).

Reply:
okay. Let's iterate over this for the next release.

Reply:
@mohitsuman dare to please +1 then?